site/utils/formUtils.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from "vitest";
+import { getDate } from "./dates";
+import { filterServices, removeDuplicateServicesByKey } from "./formUtils";
+import {
+    mockBodsServicesNoDuplicates,
+    mockBodsServicesWithDuplicates,
+    mockTndsServicesNoDuplicates,
+    mockTndsServicesWithDuplicates,
+} from "../testData/mockData";
+
+describe("filterServices", () => {
+    it.each([[mockBodsServicesNoDuplicates], [mockBodsServicesWithDuplicates]])(
```

Comment:
This test is failing now since the date has ticked over so some of the services in the mock data are now in the past, we should ideally mock the date whenever tests depend on it, something like:

```ts
import MockDate from "mockdate";
```
and then
```ts
MockDate.set("2023-08-07");

afterAll(() => {
    MockDate.reset();
});
```
in each of the top level describes would work

Reply:
fixed